src/test/java/edu/kit/compiler/lexer/LexerTest.java

```diff
@@ -48,6 +48,11 @@ public void testUnicode() throws LexException {
         var lexer = new Lexer(getIterator("int \u309e = 1;"));
         assertEquals(Keyword_Int, lexer.getNextToken().getType());
         assertThrows(LexException.class, () -> lexer.getNextToken());
+        var lexer2 = new Lexer(getIterator("int 😀 = 1;"));
```

Comment:
Same here, shouldn't this be in its own PR?

Reply:
This one didn't make it into the original lexer PR. It's technically unrelated to this fix.